src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/TilsynBarnBeregnYtelseStegIntegrationTest.kt

```diff
@@ -214,6 +220,157 @@ class TilsynBarnBeregnYtelseStegIntegrationTest(
         }
     }
 
+    @Nested
+    inner class RevurderingSkalBeholdePerioderFraForrigeBehandling {
+
+        @Test
+        fun `skal beholde perioder fra forrige behandling som er før måneden for revurderFra`() {
+            innvilgPerioderForJanuarOgFebruar(behandling.id)
+            assertHarPerioderForJanuarOgFebruar(behandling.id)
+
+            testoppsettService.oppdater(behandling.copy(status = BehandlingStatus.FERDIGSTILT))
+            val revurdering = opprettRevurdering(revurderFra = mars.atDay(15))
```

Comment:
Vet ikke hvor mye det hadde hjulpet, men er ikke så forståelig at behandlingen ferdigstilles. Kunne trukket disse to ut i en funksjon `ferdigstillBehandlingOgOpprettRevurdering(behandling, revurderFraDato)` som returnerer revurderingen? Blir to ting blandet så usikker

Reply:
Ja, mulig det burde være noe generellt for det